.github/workflows/main.yml

```diff
@@ -0,0 +1,85 @@
+name: Nar
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build-macos:
+    name: Build on macos
+    runs-on: macos-11
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install build dependencies
+      run: |
+        brew update
+        brew install automake
+    - uses: actions/setup-java@v2
+      with:
+        distribution: 'temurin'
+        java-version: '8'
+        cache: 'maven'
+    - name: Build and test the package
+      run: mvn test
+    - name: Store the build arctifacts
+      uses: actions/upload-artifact@v3
+      with:
+        name: macos-build
+        path: target/nar/louis-*
+
+  build-windows:
+    name: Build for Windows
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v2
+      with:
+        distribution: 'temurin'
+        java-version: '8'
+        cache: 'maven'
+    - name: Install build dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install texinfo libc6-dev-i386 mingw-w64
+    - name: Build and test the package
+      run: mvn test -Pcross-compile -Dhost.os=w64-mingw32 -Dos.arch=x86_64
```

Comment:
Can we build the 32-bit version in the same job or do we need a new job?

Reply:
You might be able to do it in the same job. OTOH the nice thing about separate jobs is that they run in parallel

Reply:
Build for 32 bit should be run in the same job now